test/integration/verify.test.js

```diff
@@ -14,6 +14,12 @@ const VALID_CONTENT = 'Valid content'
 const VALID_HASH = createHash(VALID_CONTENT)
 const INVALID_CONTENT = 'Invalid content'
 const EMPTY_CONTENT = ''
+const GLOS_CONTROL_FILE_CONTENT = '01'
+
+const GLOS_BATCH_BLOB_NAME = 'PENDING_FCAP_TEST_BATCH.dat'
+const GLOS_CTL_BATCH_BLOB_NAME = 'CTL_PENDING_FCAP_TEST_BATCH.ctl'
+const GLOS_CHECKSUM_BLOB_NAME = 'PENDING_FCAP_TEST_BATCH.txt'
+const GLOS_CTL_CHECKSUM_BLOB_NAME = 'CTL_PENDING_FCAP_TEST_BATCH.txt'
```

Comment:
I was thinking maybe these should be defined below the original and second variables, but above the processed variables? Like following the order of the processed variables below